app/keyboard.py

```diff
@@ -0,0 +1,106 @@
+"""
+This file will create button structures that we will use
+"""
+from telegram import InlineKeyboardButton, \
+    InlineKeyboardMarkup, ParseMode, KeyboardButton, ReplyKeyboardMarkup, ReplyKeyboardRemove
+
+"""
+Buttons from the bottom menu
+"""
+BUTTON1_ADD_DEBT = "ADD DEBT"
```

Comment:
Is it a label, identifier, or something else? It's not clear from the variable name.

Reply:
The same in the variables below

Reply:
What about this comment?